routes/_components/profile/AccountProfileMeta.html

```diff
@@ -0,0 +1,132 @@
+<div class="account-profile-meta">
+   {@html meta}
+</div>
+<style>
+  .account-profile-meta {
+    display: grid;
+    grid-template-areas: "meta-name-1 meta-value-1"
+                         "meta-name-2 meta-value-2"
+                         "meta-name-3 meta-value-3"
+                         "meta-name-4 meta-value-4";
```

Comment:
I agree this `meta-name-1`/`-2`/`-3`/`-4` is a little messy, but it's not a big deal